ietf/nomcom/views.py

```diff
@@ -1023,24 +1024,38 @@ def view_feedback_nominee(request, year, nominee_id):
             return HttpResponseForbidden('Restricted to roles: Nomcom Chair, Nomcom Advisor')
         feedback_id = request.POST.get('feedback_id', None)
         feedback = get_object_or_404(Feedback, id=feedback_id)
-        type = request.POST.get('type', None)
-        if type:
-            if type == 'unclassified':
-                feedback.type = None
-                feedback.nominees.clear()
-                messages.success(request, 'The selected feedback has been de-classified. Please reclassify it in the Pending emails tab.')
+        submit = request.POST.get('submit', None)
+        if submit == 'download':
+            fn = f'questio-{slugify(nominee.name())}-{feedback.time.date()}.txt'
```

Comment:
Leaking the slug out here isn't necessary (it's unfortunate that the slug is truncated to start with - holdover from a much older version of django). Please change this to be questionnaire-